serve.go

```diff
@@ -131,6 +133,9 @@ func serve(args []string) error {
 		APNSPrivateKeyPath:  *flAPNSKeyPath,
 		depsim:              *flDepSim,
 		tlsCertPath:         *flTLSCert,
+		CommandWebhookURL:   *flCommandWebhookURL,
+
+		webhooksHTTPClient: http.Client{},
```

Comment:
use `http.DefaultClient` instead.

Reply:
Actually looking for docs on this global instance I ran into this https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779

So a new client with a timeout may be better in this case.